cmd/install.go

```diff
@@ -80,6 +83,7 @@ func init() {
 
 	installCmd.Flags().StringVarP(&imageArchivePath, "image-archive", "i", "", "An archive containing images")
 	installCmd.Flags().BoolVarP(&askBecomePass, "askBecomePass", "", false, "Whether or not to ask for sudo password during SSH connection.")
+	installCmd.Flags().StringVarP(&quayRoot, "quayRoot", "r", "", "The folder where quay persistent data are saved. This defaults to /etc/quay-install")
```

Comment:
Can this be updated so that `/etc/quay-install` is the default?

Reply:
yes, good point I'm fixing that